config-connector/solutions/iam/kpt/subnet/README.md

```diff
@@ -0,0 +1,67 @@
+Subnet
+==================================================
+
+# NAME
+
+  subnet
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a subnet in your desired project, and grant a specific member a role (default to roles/compute.networkUser) for accessing the subnet that just created.
+
+# CONSUMPTION
+
+  Fetch the [kpt](https://googlecontainertools.github.io/kpt/) package of the solution:
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/subnet subnet
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using the cnrm-system service account.
```

Comment:
Should mention here that a ComputeNetwork is required.

Reply:
Compute network added.